DataManagementSystem/scripts/dirac-dms-filecatalog-cli.py

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env python
-########################################################################
-# $HeadURL$
-########################################################################
-__RCSID__ = "$Id$"
+
+import sys
+
+from DIRAC import gConfig, exit as dexit
```

Comment:
well spoted ! I missed that one. In principe the rule (written somewhere) is that parseCommandLine should be the very first thing. So indeed, gConfig should be imported later